index.rst

```diff
@@ -7,652 +7,415 @@ Table of contents
-
-   Volume backup and revert </user/advanced-topics/volume-backup-revert>
-
-   Standalones and HVMs </user/advanced-topics/standalones-and-hvms>
-
-   Config files </user/advanced-topics/config-files>
-
-   Secondary storage </user/advanced-topics/secondary-storage>
-
-   RPC policies </user/advanced-topics/rpc-policy>
-
-   USB qubes </user/advanced-topics/usb-qubes>
-
-   Managing qube kernels </user/advanced-topics/managing-vm-kernels>
-
-   Salt (management software) </user/advanced-topics/salt>
-
-   GUI domain </user/advanced-topics/gui-domain>
-
-   Disposable customization </user/advanced-topics/disposable-customization>
-
-   Installing contributed packages </user/advanced-topics/installing-contributed-packages>
-
-   How to make any file persistent (bind-dirs) </user/advanced-topics/bind-dirs>
-
-   GUI configuration </user/advanced-topics/gui-configuration>
-
-   Resize disk images </user/advanced-topics/resize-disk-image>
-
-   Qubes service </user/advanced-topics/qubes-service>
-
-   How to mount a Qubes partition from another OS </user/advanced-topics/mount-from-other-os>
-
-   KDE (desktop environment) </user/advanced-topics/kde>
-
-   i3 (window manager) </user/advanced-topics/i3>
-
-   AwesomeWM (window manager) </user/advanced-topics/awesomewm>
-
-
-
-Reference
-^^^^^^^^^
-
+   :caption: Advanced topics
+
+   user/advanced-topics/how-to-install-software-in-dom0
+   user/advanced-topics/volume-backup-revert
+   user/advanced-topics/standalones-and-hvms
+   user/advanced-topics/config-files
+   user/advanced-topics/secondary-storage
+   user/advanced-topics/rpc-policy
+   user/advanced-topics/usb-qubes
+   user/advanced-topics/managing-vm-kernels
+   user/advanced-topics/salt
+   user/advanced-topics/gui-domain
+   user/advanced-topics/disposable-customization
+   user/advanced-topics/installing-contributed-packages
+   user/advanced-topics/bind-dirs
+   user/advanced-topics/gui-configuration
+   user/advanced-topics/resize-disk-image
+   user/advanced-topics/qubes-service
+   user/advanced-topics/mount-from-other-os
+   user/advanced-topics/kde
+   user/advanced-topics/i3
+   user/advanced-topics/awesomewm
+
+
+.. _reference:
 
 .. toctree::
    :maxdepth: 1
+   :caption: Reference
 
-   Command-line tools </user/reference/tools>
-
-   Glossary </user/reference/glossary>
+   user/reference/tools
+   user/reference/glossary
 
 
-
-Project Security
-^^^^^^^^^^^^^^^^
-
+.. _project-security:
 
 .. toctree::
    :maxdepth: 1
+   :caption: Project Security
 
-   Qubes OS project security center </project-security/security>
-
-   Qubes security pack (qubes-secpack) </project-security/security-pack>
-
-   Verifying signatures </project-security/verifying-signatures>
-
+   project-security/security
+   project-security/security-pack
+   project-security/verifying-signatures
 
 
 Developer Documentation
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-
 Core documentation for Qubes developers and advanced users.
 
-General
-^^^^^^^
-
```

Comment:
The section name in the oirignal is General not Developers - General. if the section is named _general? similar to the rest subsections in the Developer Documentation

Reply:
This is intentional, so the TOC on the left is clearer, otherwise it easy to miss which sections belong to developers and which to users documentation.